msgpack/_unpacker.pyx

```
@@ -20,6 +19,7 @@ from cpython.exc cimport PyErr_WarnEx
 cdef extern from "Python.h":
     ctypedef struct PyObject
     cdef int PyObject_AsReadBuffer(object o, const void** buff, Py_ssize_t* buf_len) except -1
+    cdef int PyObject_GetBuffer(object o, Py_buffer *view, int flags) except -1
```

> Exceptions were ignored

Why?

https://github.com/cython/cython/blob/970c2fc0e676ca22016e14147ada0edba937dc6b/Cython/Includes/cpython/buffer.pxd#L31
Cython's declaration has `except -1`.  I don't know why exception were ignored.